zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -156,8 +155,7 @@ paths:
           required: true
           description: The key of the job.
           schema:
-            type: integer
-            format: int64
+            type: string
```

Comment:
❓ Just realizing this now: should we stick to numbers in 8.6 at least to not introduce a breaking change there? Fine for 8.7 to move to string, of course, as discussed (cannot find anymore where we discussed this 😆)